webpack.config.js

```diff
@@ -49,15 +52,38 @@ module.exports = {
     // See: https://github.com/webpack/webpack/issues/7417
     publicPath: "/",
   },
+  optimization: {
```

Comment:
I'm not sure this is the optimal way to bundle split vendor packages. I don't think this has as much impact on performance as further code-splitting, so if this is too complex or difficult to maintain we can revert it.